utils/constants.jsx

```diff
@@ -293,6 +294,7 @@ export const ModalIdentifiers = {
 };
 
 export const UserStatuses = {
+    OUT_OF_OFFICE: 'ooo',
```

Comment:
This should be added as well to [UserStatusesWeight](https://github.com/mattermost/mattermost-webapp/blob/master/utils/utils.jsx#L1132) which is used for user order in some UIs. 
Maybe `ooo: 3` then `offline: 4` but @esethna can confirm.

Reply:
They can be equal, since they have the same icon